backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubUploaderService.java

```diff
@@ -287,6 +287,7 @@ private void uploaderTask(Supplier<List<TestEvent>> testEventSupplier, boolean w
       _testEventReportingService.markTestEventsAsReported(new HashSet<>(eventsReported));
     } catch (RestClientException | UnexpectedIOException err) {
       tracking.ifPresent(upload -> _trackingService.markFailed(upload, _resultJson, err));
+      return false;
```

Comment:
Note that while this _does_ short-circuit the code below, it's just the slack hook for reporting on uploads – which isn't used anymore anyway. I just don't want to delete it bc I'm going to try to delete this whole file next week.